projects/RPi2/filesystem/usr/lib/systemd/system/serial-console.service

```diff
@@ -5,7 +5,8 @@ ConditionKernelCommandLine=console
 
 [Service]
 WorkingDirectory=/storage
-ExecStart=/bin/sh -c '. /etc/profile; exec /bin/sh'
+Environment="ENV=/etc/profile"
+ExecStart=/bin/sh -c 'exec /bin/sh'
```

Comment:
assuming that this is runtime tested and ENV= works in ash. "ExecStart=/bin/sh" should be enough. no need to re-exec